### DISCUSSION TITLE
SQLSTATE[08006] [7] could not translate host name "freshrss-db" to address: Name or service not known

### DISCUSSION THREAD

**Discussion by mariobreskic** [FreshRSS/FreshRSS#5666] [Q&A]
After updating the docker image, going to the service’s URL returns this 500 error.

To fix it, I did the following:

`docker network connect freshrss-network freshrss`

where ‘freshrss-network’ is the name of the network the database lives in (in my case “freshrss-db”), and ‘freshrss’ is the name of docker container currently running.

I hope that this will help someone else. Simply naming the new updated container the same as the old one does not prevent this error. In a way, it causes it.

The final clue came to me from https://roman-academy.medium.com/deploy-freshrss-with-docker-acf79e64bee#bd55

**Comment by Alkarex** [comment]
The article looks like a copy of our Docker readme :-)
https://github.com/FreshRSS/FreshRSS/blob/edge/Docker/README.md#create-an-isolated-network

**Reply by mariobreskic** [reply]
You are right, it is!
